vcsi/vcsi.py

```diff
@@ -948,6 +948,7 @@ def mxn_type(string):
     try:
         split = string.split("x")
         m = int(split[0])
+        assert (m > 0)
```

Comment:
Raising an `argparse.ArgumentTypeError` sounds more appropriate in this case. Also since this makes sure that `m` is positive it would be better to check that `n` is also positive.